src/service/core-services.js

```diff
@@ -123,15 +124,23 @@ function installAmpdocServicesInternal(ampdoc, isEmbedded) {
   isEmbedded
     ? adoptServiceForEmbedDoc(ampdoc, 'history')
     : installHistoryServiceForDoc(ampdoc);
-  isEmbedded
-    ? adoptServiceForEmbedDoc(ampdoc, 'resources')
-    : installResourcesServiceForDoc(ampdoc);
-  isEmbedded
-    ? adoptServiceForEmbedDoc(ampdoc, 'owners')
-    : installOwnersServiceForDoc(ampdoc);
-  isEmbedded
-    ? adoptServiceForEmbedDoc(ampdoc, 'mutator')
-    : installMutatorServiceForDoc(ampdoc);
+  if (FIE_RESOURCES) {
+    isEmbedded
```

Comment:
Nit: for the new branch can we get out of the "embedded" terminology? It makes more sense in the special-casing of amp embed.

E.g. instead of `isEmbedded`, we can do a check for: `ampdoc.isA4a() || ampdoc.isInabox()`

Reply:
I've been actually doing the opposite: moving toward `embed` terminology for all of them. Ampdoc-fie project started it, and this PR can be thought as a continuation. Basically no matter what kind of embed we're dealing with, I can't quite think of a case where we would rather share resource management with the main document. It's just not as safe. And the less delta is between inabox and a4a the better too, imho.